uk.co.compucorp.civicrm.tasksassignments/js/test/controllers/modal/modal-document_test.js

```diff
@@ -79,6 +93,55 @@ define([
       });
     });
 
+    describe('addAssignee()', function () {
+      beforeEach(function () {
+        initController();
+        addAssignee(sampleAssignee);
+      });
+
+      it('adds contact of id 5 as assignee for document', function () {
+        expect($scope.document.assignee_contact_id[0]).toBe(sampleAssignee.id);
+      });
+
+      it('adds contact of id 5 in contact search list', function () {
+        expect($rootScope.cache.contact.arrSearch[0].id).toEqual(sampleAssignee.id);
+        expect($rootScope.cache.contact.arrSearch[0].label).toEqual(sampleAssignee.label);
+      });
+    });
+
+    describe('removeAssignee()', function () {
+      beforeEach(function () {
+        initController();
+        addAssignee(sampleAssignee);
+      });
+
+      it('removes assignee form the list of assignees', function () {
+        expect($scope.document.assignee_contact_id.length).toEqual(1);
+        $scope.removeAssignee(0);
+        expect($scope.document.assignee_contact_id.length).toEqual(0);
+      });
+    });
+
+    describe('remindMeInfo()', function () {
+      beforeEach(function () {
+        initController();
+
+        spyOn(CRM,'help');
+
+        $scope.remindMeInfo();
+      });
+
+      it('makes calls to CRM.help() to display help message', function () {
+        expect(CRM.help).toHaveBeenCalled();
+        expect(CRM.help).toHaveBeenCalledWith('Remind me?', $scope.remindMeMessage, 'error');
```

Comment:
toHaveBeenCalledWith implies toHaveBeenCalled is called so toHaveBeenCalledWith is enough